python/pyarrow/_plasma.pyx

```diff
@@ -344,6 +347,35 @@ cdef class PlasmaClient:
                                                 data.get().mutable_data(),
                                                 data_size)
 
+    def create_and_seal(self, ObjectID object_id, c_string data,
```

Comment:
Should `data` be a `std::string` or rather something that exports the buffer protocol (then memory allocation not needed here)?